src/contracts/Avalanche-Parameter-Rec-21-02-2023/Avalance-Parameter-Rec-21-02-2023.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import '../common/StewardBase.sol';
+import {AaveV3Avalanche} from 'aave-address-book/AaveV3Avalanche.sol';
+
+/**
+ * @dev This steward sets configure reserve as collateral for Link.e and awavx on AAVE V3 Avalanche
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xbe3ff894ff9a979846b45e0fece4368245f61fa1d449d3761a3990b8da4aa6d7
+ * - Dicussion: https://governance.aave.com/t/arc-chaos-labs-risk-parameter-updates-aave-v3-avalanche-2023-02-07/11603
+ */
+contract AaveV3AvaParamsSteward is StewardBase {
+    
+    address public constant LINKe = 0x5947BB275c521040051D82396192181b413227A3;
```

Comment:
fine, but you could use assets library to make reviews slightly easier  https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Avalanche.sol#L82